Linux/sgx/libsgx_tsgxssl/tunistd.cpp

```diff
@@ -99,19 +99,40 @@ size_t sgxssl_read(int fd, void *buf, size_t count)
 	// In addition, the function is used by bss_sock.c as readsocket function.
 	// It is unreachable under the assumption that TLS support is not required.
 	// Otherwise should be implemented as OCALL.
-	size_t retval = 0;
+	ssize_t retval = 0;
 	sgx_status_t ret = u_sgxssl_read(&retval, fd, buf, count);
 	if (ret != SGX_SUCCESS)
 	{
 		FEND;
-		return 0;
+		return -1;
 	}
 	FEND;
 
 	return retval;
 }
 
-// TODO
+int sgxssl_open(const char *filename, int flags)
+{
+	FSTART;
+
+    if (filename == NULL) return -1;
+	// In addition, the function is used by b_sock2.c as closesocket function.
+	// It is unreachable under the assumption that TLS support is not required.
+	// Otherwise should be implemented as OCALL.
+	int retval = 0;
+	sgx_status_t ret = 
```

Comment:
Suggest to write the code of line 123 and 124 in one line.

Reply:
updated